File: 0330PM-0500PM_Data_Analytics/13th_May/if_else.py
# Conditional Statements.
# if , if_else , Nested if_else , elif, Complex Cond.
# Looping :- for , while

"""
if
Syntax:-
if condition:
    statements

# Example

if 10>50:
    print("Hello India")

if_else
Syntax:-
if condition :
    statements
else :
    statements
    

# Example
if 10>50:
    print("Hello India")
else:
    print("Hello World")



# WAP to check an entered number is even or odd.

num = int(input("Enter A Number : "))

if num%2 == 0:
    print("Even")
else:
    print("Odd")



# WAP to find greater value between two.

a = int(input("Enter A Number : "))
b = int(input("Enter B Number : "))
if b>a:
    print(b)
else:
    print(a)
    
"""

# Nested if_else
"""
syntax:-
if condition:
    if condition:
        true Statement
    else:
        false Statements
else:
    if condition:
        true statement
    else:
        true statement


# WAP to check an entered number is positive,
# negative or zero.

num = int(input("Enter A Number : "))
# 0
if num>0:
    print("Positive")
else:
    if num<0:
        print("Negative")
    else:
        print("Zero")



# WAP to check an entered character is vowel or not.
# a , e , i , o , u
ch = input("Enter A Character : ") 
if ch=='a': 
    print('Vowel')
else:
    if ch=='e':
        print("Vowel")
    else:
        if ch=='i':
            print("Vowel")
        else:
            if ch=='o':
                print("Vowel")
            else:
                if ch=='u':
                    print("Vowel")
                else:
                    print("Consonant")

# elif
Syntax:-
if condition:
    statement
elif condition:
    statement
else:
    statement


# WAP to check an entered character is vowel or not.
# a , e , i , o , u
ch = input("Enter A Character : ") 
if ch=='a': 
    print('Vowel')
elif ch=='e':
    print("Vowel")
elif ch=='i':
    print("Vowel")
elif ch=='o':
    print("Vowel")
elif ch=='u':
    print("Vowel")
else:
    print("Consonant")


# complex condition
# WAP to check an entered character is vowel or not.
# a , e , i , o , u

ch = input("Enter A Character : ")
if ch=='a' or ch=='e' or ch=='i' or ch=='o' or ch=='u':
    print("Vowel")
else:
    print("Consonant")


ch = input("Enter A Character : ")
if ch in "aeiouAEIOU":
    print("Vowel")
else:
    print("Consonant")

 """   

File: 10AM-1130AM_MERN_STACK/JavaScript/15th_May_JS_Conditional_Statements/if_else.js
/*
Control Statements:-
if , if_else , Nested if_else , complex condition
Looping:- for , while , do_while

IF 
Syntax:-
if ( condition ){
    statements;
}

if (10>7){
    console.log("Hello"); 
}
if (10>70){
    console.log("Hello"); 
}

IF_ELSE
if (condition){
    true_Statements
}else{
    false_Statements
}
if(10>70){
    console.log("HELLO");
}else{
    console.log("BYE");
}

// WAP to check an entered number is even / odd.
var num = 97;
if (num%2 == 0){
    console.log("Even");
}else{
    console.log("Odd");
}
   
// WAP to find greater value between two.
var a = 10;
var b = 20;
if (a>b){
    console.log(a);
}else{
    console.log(b);
} 


Nested_IF_ELSE
if (condition){
    if(condition){
        true
    }else{
        false    
    }
}else{
    if(condition){
        true
    }else{
        false
    }
}



// WAP to check an entered number is positive, negative or zero.
// num>0  -> positive , num<0 -> negative , num==0 -> zero
var num = 0;
if (num>0){
    console.log("Positive");
}else{
    if(num<0){
        console.log("Negative");
    }else{
        console.log("Zero");
    }
}



// WAP to check an entered charcter is vowel or consonants.
// Vowel:- a,e,i,o,u
var ch = 'z';
if(ch=='a')
    console.log("Vowel");
else
    if(ch=='e')
        console.log("Vowel");
    else
        if(ch=='i')
            console.log("Vowel");
        else
            if(ch=='o')
                console.log("Vowel")
            else
                if(ch=='u')
                    console.log("Vowel")
                else
                    console.log("Consonant");



Complex_Condition  -> Using Logical Operator
if(condition and condition or condition){
    true
}else{
    false
}
*/

// WAP to check an entered character is vowel or consonant.
var ch = 'i';
if ( ch=='a' || ch=='e' || ch=='i' || ch=='o' || ch=='u' ){
    console.log("Vowel");
}else{
    console.log("Consonant");
}
